test/metabase/api/downloads_exports_test.clj

```diff
@@ -958,37 +957,20 @@
                     :subscription-attachment (first subscription-result)}))))))))
 
 (deftest downloads-row-limit-test
-  (testing "Downloads row limit works."
-    (mt/with-temporary-setting-values [public-settings/download-row-limit 105]
+  (testing "Downloads row limit respects minimum (#52019)"
+    (mt/with-temporary-setting-values [public-settings/download-row-limit 950]
```

Comment:
https://metaboat.slack.com/archives/CKZEMT1MJ/p1733555640737189

Reply:
Why don't you make that `1048576` a var so you can `with-redefs` it so this test can still check just ~100 rows instead having to make this test super slow again